SinaiStCatherine/SinaiEt007.xml

```diff
@@ -0,0 +1,300 @@
+                     <title type="complete">Prayers for the weekdays</title>
+                     <textLang mainLang="gez"/>
+           </msItem>
+                 
+                 <msItem xml:id="ms_i2">
+                   <locus from="60r" to="64v"/>
+                   <title type="complete" ref="LIT4154Prayer"/>
+                   <textLang mainLang="gez"/>
+                 </msItem>
+                 
+                 <msItem xml:id="ms_i3">
+                   <locus from="65r" to="70v"/>
+                   <title type="complete">Tamāḫḍǝno śǝllāse</title>
+                   <textLang mainLang="gez"/>
+                 </msItem>
+                 
+                 <msItem xml:id="ms_i4">
+                   <locus from="70v" to="72v"/>
+                   <title type="complete">Ṭǝʿuma sǝm</title>
+                   <textLang mainLang="gez"/>
+                 </msItem>
+                 
+                 <msItem xml:id="ms_i5">
+                   <locus from="72v" to="128v"/>
+                   <title type="complete" ref="LIT2296Sayfas"/>
+                   <textLang mainLang="gez"/>
+                 </msItem>
+                 
+               </msContents>
+              
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf" quantity="128">128</measure>
+                        <measure unit="quire" quantity="14">14</measure>
+                          <measure unit="g" type="weight">405</measure>
+                           <dimensions type="outer" unit="mm">
+                              <height>140</height>
+                           <!--   <width></width>-->
+                             <depth>45</depth> 
+                             <!--The depth is measured accross the spine. The other side of the manuscript is swollen to roughly 9 cm.-->
+                           </dimensions>
+                          <dimensions type="leaf" unit="mm" xml:id="leafdim">
+                            <height>130</height>
+                            <width>84</width>
+                          </dimensions>
+                          <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#39r"/>.</note>
+                        </extent>
+                        <foliation>Foliation in the upper right corner of each folio, carried out in February 2023.</foliation>
+                       
+                        <collation>                      
+                          <signatures>On several quires, small undecorated quire marks in the upper inner corner of the first folio.
+                          It is possible that quire marks had originally been present in every quire, but are not visible everywhere anymore.</signatures>
```

Comment:
Should be checked in the manuscript, if there the quire marks are usually visible. It is not uncommon that quire marks were indicated not in every quire.

Reply:
As I wrote, I don't have the images now